packages/lib/src/doc-utils/validation.ts

```
@@ -0,0 +1,192 @@
+}
+
+export const processFile = async (
+	file: File,
+	setIsProcessing: (value: boolean) => void,
+	setProgress: (value: number) => void,
+	setValidationErrors: (errors: string[]) => void,
+	Tesseract: {
+		recognize: (
+			arg0: File,
+			arg1: string,
+			arg2: { logger: (message: any) => void },
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)